samples/KvsAudioOnlyStreamingSample.c

```diff
@@ -0,0 +1,264 @@
+// UINT16 gKeyFrameCount = 0;
+
+typedef struct {
+    PBYTE buffer;
+    UINT32 size;
+} FrameData, *PFrameData;
+
+typedef struct {
+    volatile ATOMIC_BOOL firstVideoFramePut;
+    UINT64 streamStopTime;
+    UINT64 streamStartTime;
+    STREAM_HANDLE streamHandle;
+    CHAR sampleDir[MAX_PATH_LEN + 1];
+    FrameData audioFrames[NUMBER_OF_AUDIO_FRAME_FILES];
+    BOOL firstFrame;
+    UINT64 startTime;
+} SampleCustomData, *PSampleCustomData;
+
+PVOID putAudioFrameRoutine(PVOID args)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    PSampleCustomData data = (PSampleCustomData) args;
+    Frame frame;
+    UINT32 fileIndex = 0;
+    STATUS status;
+    UINT64 runningTime;
+
+    CHK(data != NULL, STATUS_NULL_ARG);
+
+    frame.frameData = data->audioFrames[fileIndex].buffer;
+    frame.size = data->audioFrames[fileIndex].size;
+    frame.version = FRAME_CURRENT_VERSION;
+    frame.trackId = DEFAULT_AUDIO_ONLY_TRACK_ID;
+    frame.duration = 0;
+    frame.decodingTs = 0;     // relative time mode
+    frame.presentationTs = 0; // relative time mode
+    frame.index = 0;
+    frame.flags = fileIndex % DEFAULT_KEY_FRAME_INTERVAL == 0 ? FRAME_FLAG_KEY_FRAME : FRAME_FLAG_NONE;
+
+    while (defaultGetTime() < data->streamStopTime) {
+        status = putKinesisVideoFrame(data->streamHandle, &frame);            
+        if (STATUS_FAILED(status)) {
+            printf("putKinesisVideoFrame for audio failed with 0x%08x\n", status);
+            status = STATUS_SUCCESS;
+        }
+
+        frame.presentationTs += SAMPLE_AUDIO_FRAME_DURATION;
+        frame.decodingTs = frame.presentationTs;
+        frame.index++;
+
+        fileIndex = (fileIndex + 1) % NUMBER_OF_AUDIO_FRAME_FILES;
+        frame.flags = fileIndex % DEFAULT_KEY_FRAME_INTERVAL == 0 ? FRAME_FLAG_KEY_FRAME : FRAME_FLAG_NONE;
+        frame.frameData = data->audioFrames[fileIndex].buffer;
+        frame.size = data->audioFrames[fileIndex].size;
+
+        // synchronize putKinesisVideoFrame to running time
+        runningTime = defaultGetTime() - data->streamStartTime;
+        if (runningTime < frame.presentationTs) {
+            THREAD_SLEEP(frame.presentationTs - runningTime);
+        }
+    }
+
+CleanUp:
+
+    if (retStatus != STATUS_SUCCESS) {
+        printf("putAudioFrameRoutine failed with 0x%08x", retStatus);
+    }
+
+    return (PVOID)(ULONG_PTR) retStatus;
+}
+
+INT32 main(INT32 argc, CHAR* argv[])
+{
+    PDeviceInfo pDeviceInfo = NULL;
+    PStreamInfo pStreamInfo = NULL;
+    PClientCallbacks pClientCallbacks = NULL;
+    PStreamCallbacks pStreamCallbacks = NULL;
+    CLIENT_HANDLE clientHandle = INVALID_CLIENT_HANDLE_VALUE;
+    STREAM_HANDLE streamHandle = INVALID_STREAM_HANDLE_VALUE;
+    STATUS retStatus = STATUS_SUCCESS;
+    PCHAR accessKey = NULL, secretKey = NULL, sessionToken = NULL, streamName = NULL, region = NULL, cacertPath = NULL;
+    UINT64 streamStopTime, streamingDuration = DEFAULT_STREAM_DURATION, fileSize = 0;
+    TID audioSendTid, videoSendTid;
```

Comment:
Let's remove all video related variables to clean up this sample so `videoSendTid` which is the thread id for that routine can be removed.